react/features/base/react/components/web/MeetingsList.js

```diff
@@ -199,3 +237,14 @@ export default class MeetingsList extends Component<Props> {
         );
     }
 }
+
+/**
+ * Maps redux state to component props.
+ *
+ * @returns {Array}
+ */
+function _mapStateToProps() {
+    return { };
+}
+
+export default translate(connect(_mapStateToProps)(MeetingsList));
```

Comment:
Do you need connect at all? Also maybe we should keep the pattern here and pass the function that deletes everything as a prop. This way meetings list will be generic.

Reply:
I changed connect function without parameter, I don't know how to call prop function (delete or dispatch) without connect. Moreover dispatch actions are useful^^

Reply:
See how we do it for onPress prop. That's what I mean. This way you won't need dispatch here. And this component will still be generic. And also it will be more consistent since we do this for onPress already.

Reply:
I don't think this one is addressed. Do you plan to address it?

Reply:
In fact, I have no idea to do without that... i was inspired by the function deleteRecentListEntry() which is available on mobile to delete one entry. This function use dispatch and connect (to translate and dispatch). I also add this function the ReducerRegistry like other.
I you want a better solution, I can close this PR and think about a better solution with more knowledge about react.

Reply:
No no don't close it! It just needs a very very small change.

Here's more detailed explanation about the change that I propose:
 - move `_onClear` method in react/features/recent-list/components/RecentList.web.js
 - pass the `_onClear` method to the `MeetingsList` component in the render method as a `onClear` property.
 - in react/features/base/react/components/web/MeetingsList.js use the `onClear` property value to execute the function when the new button is pressed. it is good to add a check if the property is not passed to hide the button
As a result:
 - You won't need connect in react/features/base/react/components/web/MeetingsList.js at all.
Purpose of the change:
MeetingList is used to just render whatever we pass to the component. All the implementation details are provided by the components that are using it. In this case `RecentList`.

Let me know if you want to try to do the change!

If not, would you allow me to push a commit to your PR? Note that it may take a while to clear my backlog of more urgent tasks until I get to this one!

Reply:
i'm trying this ! (sorry didn't have many time today... double IT crisis due to quarantine and VPN network overload :(). I'll update asap !

Reply:
Done ! :) thx a lot for these advices and explanations !